calcom/cal.com
# chore: [app-router-migration 10.5]: replace new trpc helper with legacy ssrInit/ssgInit

## What does this PR do?
We found that in nextjs v > 13.4.6 custom server side trpc helper is no more needed. 
This PR removes usages of custom ssrInit/ssgInit. 

## Type of change



- Bug fix (non-breaking change which fixes an issue)
- Chore (refactoring code, technical debt, workflow improvements)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Tests (Unit/Integration/E2E or any other test)
- This change requires a documentation update

## How should this be tested?

Navigate to future page that use ssrInit/ssgInit, ensure that queries are prefetched (no client-side requests are sent for queries that are fetched on the server side.)

## Mandatory Tasks
